test/integration/migrator/MigratorTest.java

```diff
@@ -48,7 +48,7 @@
     private static final Path logDir = dataDir.resolve("logs");
     private static final Database options = new Database().dataDir(dataDir).logsDir(logDir);
     private static final String database = "typedb";
-    private static final Path schemaPath = Paths.get("test/integration/migrator/schema.gql");
+    private static final Path schemaPath = Paths.get("test/integration/migrator/schema.tql");
     private final Path dataPath = Paths.get("test/integration/migrator/data.typedb");
```

Comment:
we're adding this file `test/integration/migrator/data.typedb 
`, of size 6.25MB, intentionally, right?

Reply:
yes, wanted something a bit more complete